### PR TITLE
grype 0.62.3 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

continuing #96807

### CODE REVIEW

File: Formula/grype.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    ldflags = %W[
+      -s -w
+      -X github.com/anchore/grype/internal/version.version=#{version}
+      -X github.com/anchore/grype/internal/version.gitCommit=brew
+      -X github.com/anchore/grype/internal/version.buildDate=#{time.iso8601}
+    ]
+
+    # Building for OSX with -extldflags "-static" results in the error:
+    # ld: library not found for -lcrt0.o
+    # This is because static builds are only possible if all libraries
```
Comment (SMillerDev) [inline Formula/grype.rb:21]: If all libraries what? And why do we want static?

Comment (chenrui333) [reply Formula/grype.rb:21]: yeah, trying to mimic what upstream does, I can remove it.

File: Formula/grype.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class Grype < Formula
+  desc "Vulnerbility scanner for container images and filesystems"
```
Comment (nandahkrishna) [inline Formula/grype.rb:2]:
```suggestion
  desc "Vulnerability scanner for container images and filesystems"
```

**Review by nandahkrishna — COMMENTED** [review]
Minor typo.

**Comment by chenrui333** [comment]
rerun for 11-arm runner issue.

**Review by p-linnane — APPROVED** [review]

**Review by nandahkrishna — APPROVED** [review]

**Pull request merged**